moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
```

Comment:
can you add a comment here about what limited does?

Reply:
This sounds a lot like the `limited` arg that some of the `universal_robot` launch files have/use.

I don't believe it should be part of a template file introduced by this contribution.

Reply:
I did remove it as @gavanderhoorn suggestion makes sense.